src/hiero_sdk_python/client/client.py

```
@@ -2,8 +2,9 @@
 Client module for interacting with the Hedera network.
 """
 
-from typing import NamedTuple, List, Union, Optional
-
+import os
+from typing import NamedTuple, List, Union, Optional, Literal
```

Ruff (0.14.10)

`typing.List` is deprecated, use `list` instead

(UP035)